charms/worker/k8s/charmcraft.yaml

```
@@ -224,6 +224,17 @@ config:
           runtime-config=batch/v2alpha1=true profiling=true
         will result in kube-apiserver being run with the following options:
           --runtime-config=batch/v2alpha1=true --profiling=true
+    kube-apiserver-extra-sans:
+      type: string
+      default: ""
+      description: |
+        Comma-separated list of extra sans for the kube-apiserver self-signed
+        certificates.
```

Would you accept space separated?

```suggestion
        Space-separated list of extra Subject Alternative Names for the kube-apiserver self-signed
        certificate.
```

---

Sure. I see that some options expect comma separated values (``bootstrap-pod-cidr``) while others use space separation (``dns-upstream-nameservers``).